docs/zksync-error-howto.md

```diff
@@ -0,0 +1,377 @@
+# What is zksync-error?
```

Comment:
Formatting in this file is odd. Could you please make sure that it's properly formatted, e.g.:

- aligned to 120 line length
- no blocks of many empty lines
- only one H1 header